### PR TITLE
2bwm: new package at 0.3.0

### PR DESCRIPTION
**Pull request by icyphox**
## Description of package
A fast floating WM written in XCB, with the particularity of having 2 borders.

## Checklist

- [x] Latest upstream version.
- [x] I agree to maintain this package (**required**).
- [x] I agree to notify KISS if I can no longer maintain this package (**required**).

### CODE REVIEW

File: community/2bwm/version [hunk 1]
```diff
@@ -0,0 +1 @@
+0.3.0 1
```
Comment (dylanaraps) [inline community/2bwm/version]: Version should be `0.3`.

Comment (icyphox) [reply community/2bwm/version]: Should be fine now.

**Comment by dylanaraps** [comment]
:+1:

File: community/2bwm/version (hunk 1 above)
Comment (dylanaraps) [reply community/2bwm/version]: Thanks :)

**Pull request merged**